.github/actions/update-config/action.yml

```diff
@@ -0,0 +1,35 @@
+name: Update Config
+description: 'Updates the configuration'
+
+inputs:
+  host-name:
+    description: 'The host name to update'
+    required: true
+  android-facet-id:
+    description: 'The android facet id to update'
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Update Configuration (loginRequestURL)
+      uses: jossef/action-set-json-field@v2.1
+      with:
+        file: assets/config_authentication_cloud.json
+        field: login.loginRequestURL
+        value: "https://${{ inputs.host-name }}.mauth.nevis.cloud/_app/auth/pwd"
```

Comment:
What if we want to change the hardcoded `.mauth.nevis.cloud` part in the future? I think it could be the part of `inputs.host-name` as well.

Reply:
It is not needed as it is mentioned in the README as well. Since the app support only auth cloud its enough to replace only the hostname.